themes/blend/composer.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "mautic/theme-blend",
+  "description": "Mautic Blend Theme",
+  "type": "mautic-theme",
+  "keywords": ["mautic","theme"],
+  "minimum-stability": "dev",
+  "require": {
+    "mautic/core-lib": "^5.2"
```

```suggestion
    "mautic/core-lib": "^6.0"
```

---

Why require minimum version 6.0 when it’s also compatible with 5.2?

---

Because it would not be possible to install on Mautic 6. If this is compatible with Mautic 5.2, then change to:
```suggestion
    "mautic/core-lib": "^5.2 || ^6.0"
```

---

Hmm got it, we could set >=5.2 then, to allow anything above 5.2, including 6.x

What do you think?

---

Either way works, `^5.2 || ^6.0` or `>=5.2`. The latter is more convenient, as the version constraint only needs to be updated in case of a BC break.

```suggestion
    "mautic/core-lib": ">=5.2"
```